From 03e10eabcd7b72c3730ca75ffe5276db2b7de1b6 Mon Sep 17 00:00:00 2001
From: al <al@b3059339-0415-0410-9bf9-f77b7e298cf2>
Date: Tue, 24 Dec 2019 10:41:24 +0000
Subject: [PATCH 1/5] codecs.conf: Add AV1 decoder dav1d via FFmpeg

Thanks to

* Andrew-R (in ticket #2355)
* Sylvain (on mplayer-dev-eng ml)
* Carl Eugen Hoyos (in private communication)

You all provided helpful patches, suggestions and comments!

Fixes part of ticket #2355



git-svn-id: svn://git.mplayerhq.hu/mplayer/trunk@38155 b3059339-0415-0410-9bf9-f77b7e298cf2
---
 etc/codecs.conf | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/etc/codecs.conf b/etc/codecs.conf
index 8b3db9797..a0757c79d 100644
--- a/etc/codecs.conf
+++ b/etc/codecs.conf
@@ -8,6 +8,16 @@ release 20111012
 ;=============================================================================
 ;                   VIDEO CODECS
 ;=============================================================================
+videocodec fflibdav1d
+  info "FFmpeg wrapper for libdav1d"
+  status working
+  fourcc AV01,av01
+  driver ffmpeg
+  dll libdav1d
+  out YV12,420P10,420P12
+  out 422P,422P10,422P12
+  out 444P,444P10,444P12
+  out GBR24P,GBR10P,GBR12P
 
 videocodec ffiff
   info "FFmpeg IFF ANIM/ILBM/PBM/RGB8/RGBN"
-- 
2.26.0

